### PR TITLE
Implemented creating hubspot contacts when user use contact us form.

### PR DESCRIPTION
**Pull request by dmytropanontko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "classnames": "^2.2.5",
     "express": "^4.16.2",
     "express-uncapitalize": "^1.0.0",
+    "https": "^1.0.0",
```
Comment (epaminond) [inline package.json]: @dmytropanontko , could we just use regular [fetch](https://developer.mozilla.org/ru/docs/Web/API/Fetch_API/Using_Fetch) for our purposes?

Comment (dmytropanontko) [reply package.json]: @epaminond I think yes, but need to check

File: server.js [hunk 2]
```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
...
+    phone: phone.value.toString(),
+  });
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+      'Content-Length': postData.length,
+    },
+  };
+  const hubUrl = `https://forms.hubspot.com/uploads/form/v2/2769865/8552b6b3-5f76-4f18-a390-48b8f418fedc?${postData}`;
```
Comment (epaminond) [inline server.js]: @dmytropanontko , could you extract hubspot user-data to `private.config.js`?

File: server.js [hunk 3]
```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
...
+
+  fetch(hubUrl, options)
+    .then(checkStatus)
+    .then(response => console.log(response))
```
Comment (epaminond) [inline server.js]: @dmytropanontko , what will be output here? Is it usefule?

File: server.js [hunk 4]
```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
 
 const handle = app.getRequestHandler();
 
+const checkStatus = response => response.ok ? response.json() : Promise.reject(response.json());
```
Comment (epaminond) [inline server.js]: Could it be an arrow-function?

File: server.js [hunk 5]
```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
...
+
+function sendContactToHubSpot({ firstname, lastname, email, phone }) {
```
Comment (epaminond) [inline server.js]: @dmytropanontko , could it be an arrow-function?

File: server.js [hunk 6]
```diff
@@ -112,12 +138,13 @@ app.prepare().then(() => {
...
         throw err;
       }
-
       res.send({
         errorField: {},
         status: 'Message sent',
       });
     });
+
+    sendContactToHubSpot({ firstname, lastname, email, phone });
```
Comment (epaminond) [inline server.js]: @dmytropanontko  you'd rather pass actual value here to keep function interface clean. E.g. like this:
```
sendContactToHubSpot({ firstname: firstName.value, lastname: lastName.value, email: email.value, phone: phone.value.toString() });
```

File: server.js (hunk 5 above)
Comment (dmytropanontko) [reply server.js]: @epaminond yes

File: server.js (hunk 3 above)
Comment (dmytropanontko) [reply server.js]: @epaminond I forgot remove that

File: server.js (hunk 6 above)
Comment (dmytropanontko) [reply server.js]: @epaminond okey

**Pull request merged**